### PR TITLE
fix: optimise build and coverage

### PR DESCRIPTION
**Pull request by sanpj2292**
## What are the changes introduced in this PR?

We want to remove executing tests multiple times.
Suppress logging.

## What is the related Linear task?

Resolves INT-XXX

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Review by koladilip — APPROVED** [review]

**Review by koladilip — DISMISSED** [review]

File: .github/workflows/test.yml
Comment (krishna2020) [inline .github/workflows/test.yml:1]: should we rename this file to ut-test.yml then ?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -26,19 +40,6 @@ jobs:
       - name: Install Dependencies
         run: npm ci
 
-      - name: Run Unit Tests
-        run: |
-          npm run test:js:ci
```
Comment (krishna2020) [inline .github/workflows/test.yml:31]: where would others tests reside if removed from here ?

Comment (koladilip) [reply .github/workflows/test.yml:31]: transformer tests covered in code coverage yaml

Comment (krishna2020) [reply .github/workflows/test.yml:31]: Then why to remove the parallelism ?
Let the code coverage action run independently which covers execution of all the tests.
And this action - good with removing duplicate unit tests & restrict it only to UserTransformation tests & rename the file/action to ut-tests.

Comment (koladilip) [reply .github/workflows/test.yml:31]: for coverage workflow also we are running tests that's why we remove from this

Comment (koladilip) [reply .github/workflows/test.yml:31]: addressed

**Review by koladilip — APPROVED** [review]

**Review by koladilip — APPROVED** [review]

**Pull request merged**